apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -685,6 +693,18 @@ function getCloseButtonUrl( calypsoPort ) {
 
 		window.wp.hooks.doAction( 'updateCloseButtonOverrides', data );
 	};
+
+	addFilter(
+		'a8c.WpcomBlockEditorNavSidebar.closeUrl',
+		'wpcom-block-editor/getCloseButtonUrl',
```

Comment:
A tiny nit, the function names in the `addAction` calls you've used elsewhere have an `a8c/` prefix, but for these filters it's just `wpcom-block-editor/`.

Reply:
I think `wpcom-` felt like it's own vendor prefix in a way. But I see basically every namespace arg in Gutenberg uses a vendor prefix so I'll put an `a8c` on mine too.

Reply:
Actually yeah, even just for consistency because as you point out in other places I'm doing `a8c/wpcom-block-editor/closeEditor`